addon/components/bsy-icon.js

```diff
@@ -0,0 +1,38 @@
+import Ember from 'ember';
+
+const { Component, computed } = Ember;
+
+const BsyIconComponent = Component.extend({
+  classNameBindings: ['iconClass', 'iconSize', 'iconType', 'fixedWidth:icon-fw'],
+
+  name: null,
```

Comment:
If we expect to have those values as a string we should specify it by assigning an empty string `''`